src/test/java/ro/unibuc/hello/controller/GreetingsControllerIntegrationTest.java

```diff
@@ -53,7 +50,7 @@ public static void tearDown() {
 
     @DynamicPropertySource
     static void setProperties(DynamicPropertyRegistry registry) {
-        final String MONGO_URL = "mongodb://root:example@localhost:";
+        final String MONGO_URL = "mongodb://localhost:";
```

Comment:
For some reason it seems the testcontainers version of Mongo was ignoring the `MONGO_INITDB_ROOT_*` env vars above and connecting with user/password did not work.